docs/tabpy-tools.md

```diff
@@ -136,6 +137,95 @@ client.remove('WillItDefault')
 
 ```
 
+## Predeployed Functions
+
+
+To setup models download the latest version of TabPy and follow the instructions
+to install and start up your server. Once your server is running, navigate to the
+models directory and run setup.py.  If your TabPy server is running on the default
+port (9004), you do not need to specify a port when launching the script. If your
+server is running on a port other than 9004 you can specify a different port in
+the command line like so
+
+```python
+
+python setup.py 4047 
+
+```
+
+The setup file will install all of the necessary dependencies (sklearn, nltk,
+textblob, pandas, & numpy) and deploy all of the prebuilt models located in
+./models/scripts. For every model that is successfully deployed a message will
```

Comment:
For folders and file names same as function and package names I suggest we surround them with ` quotes.